packages/clay-form/src/DualListBox.tsx

```diff
@@ -60,6 +60,16 @@ interface IProps extends React.HTMLAttributes<HTMLDivElement> {
 		transferRTL: string;
 	};
 
+	/**
+	 * Disables the button responsible for moving items from right to left box.
+	 */
+	disableRTL: boolean;
```

Comment:
These should be optional params `disableRTL?:`

Reply:
https://github.com/liferay/clay/pull/3487#discussion_r453835692